packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/scim/test-instance.md

```diff
@@ -21,55 +21,96 @@
+      Repeat this step until you remove all the mappings for the attributes that you want to delete.
+   4. Select **To App** on the left **Settings** panel.
+   5. Scroll to the **Okta Attribute Mappings** section.
+   6. Click **X** next to the attribute that you want to delete, and then click **OK** to confirm. 
+   7. Repeat this step until you remove all the mappings for the attributes that you want to delete.
 
-        Repeat this step until you remove all the mappings for the attributes that you want to delete.
+**Delete attributes**
 
-     1. After removing all the mappings for the attributes that you want to delete, click **Go to Profile Editor**.
+   1. After removing all the mappings for the attributes that you want to delete, click **Go to Profile Editor**.
 
-     1. In the Profile Editor, delete all the corresponding attributes from the mapping by clicking **X** next to the attribute and then **Delete Attribute** to confirm.
+   2. In the **Profile Editor**, delete all the corresponding attributes from the mapping by clicking **X** next to the attribute and then **Delete Attribute** to confirm.
+      Repeat this step for all the attributes that you want to delete.
 
-        Repeat this step for all the attributes that you want to delete.
+**Add attributes and mappings**
 
-   * Add attributes:
+   1. In the **Profile Editor**, click **Add Attribute**.
 
-     1. In the Profile Editor, click **Add Attribute**.
+   2. Enter the information for the new attribute that you’re adding and then click **Save**.
+      > **Note:** The **Scope** property determines whether the attribute that you're adding can be assigned at a group level or per user. If you want your admins to assign a value for this attribute at a group level, don't select the **User personal** checkbox.
 
-     1. Enter the information for the new attribute that you’re adding and then click **Save**.
+   3. After adding attributes, go back to the **{yourApp} Attribute Mappings** section and click **Edit** to map your new attributes. A dialog appears with two dropdown fields.
 
-        > **Note:** The **Scope** property determines whether the attribute that you're adding can be assigned at a group level or per user. If you want your admins to assign a value for this attribute at a group level, don't select the **User personal** checkbox.
+   <div class="three-quarter border">
 
-     1. After adding attributes, go back to the **{yourApp} Attribute Mappings** section and click **Edit** to map your new attributes. A dialog appears with two dropdown fields.
+   ![Displays the map attribute dialog.](/img/oin/scim_check-attributes-14.png)
 
-     1. Select **Map from Okta Profile** in the first dropdown list.
-     1. In the second dropdown list, select the Okta profile attribute that you want to map over to the SCIM attribute.
-     1. Click **Save**.
+   </div>
 
-         Repeat these steps for all SCIM attributes that you want to map (from Okta to your app).
+=======
```

Comment:
I don't recognize this. Feel free to ignore if this is intentional markdown but wanted to call it out to be sure